catboost/python-package/catboost/_catboost.pyx

```diff
@@ -3859,6 +3859,21 @@ cdef class _PoolBase:
 
 
     cpdef save_quantization_borders(self, output_file):
+        """
+        Save file with borders used in numeric features binarization.
```

Comment:
`binarizarion` -> `quantization`.
`binarization` is sometimes used in the code but this term is incorrect for this process.